pyresample/geometry.py

```diff
@@ -802,8 +799,8 @@ class AreaDefinition(BaseDefinition):
         Pixel width in projection units
     pixel_size_y : float
         Pixel height in projection units
-    pixel_upper_left : list
-        Coordinates (x, y) of center of upper left pixel in projection units
+    top_left_extent : list
+        Coordinates (x, y) of extent of upper left pixel in projection units
```

Comment:
I think `center` is an important word here, it shouldn't be removed

Reply:
After consulting with @djhoese, we decided it was better to use the top-left corner of the top-left pixel for area definitions, rather than the center of the top-left pixel. Hence, leaving "center" in the description would be incorrect.